src/test/scala/com/redis/SerializationSpec.scala

```diff
@@ -25,16 +26,18 @@ class SerializationSpec extends FunSpec
 
     r.hmget("hash", "field1", "field2") should be(Some(Map("field1" -> "1", "field2" -> "2")))
 
-    implicit val parseInt = Parse[Int](new String(_).toInt)
+    {
```

Comment:
without these brackets, the earlier call to r.hmset will error in scala 3.0.2 with a message about an incorrect forwards-reference to parseInt